docs/Reference/genesis.md

```diff
@@ -1,48 +1,92 @@
+|---------------------|-:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Milestone blocks    | [Milestone blocks for the network](#milestone-blocks).                                                                                                                                      |
+| `chainId`           | [Chain ID for the network](../Concepts/NetworkAndChainID.md).                                                                                                                               |
+| `istanbul`          | Specifies network uses [IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md) or [QBFT](../HowTo/Configure/Consensus-Protocols/QBFT.md) and contains [IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) or [QBFT configuration items](../HowTo/Configure/Consensus-Protocols/QBFT.md#genesis-file), respectively. |
+| `clique`            | Specifies network uses [Clique](../HowTo/Configure/Consensus-Protocols/Clique.md) and contains Clique configuration items.                                                                  |
+| `txnSizeLimit`      | Maximum transaction size. The default is `64` (kilobytes), increased from Ethereum's default `32`. This is configurable up to `128`.                                                        |
+| `maxCodeSize`       | Maximum smart contract code size. The default is `32` (kilobytes), increased from Ethereum's default `24`. This is configurable up to `128`.                                                |
+| `isQuorum`          | Set to `true` to allow interoperable private transactions between GoQuorum and [Hyperledger Besu](https://besu.hyperledger.org/en/stable/) using the Tessera private transaction manager.   |
+
+!!! note
+
+    Using the [Raft consensus protocol](../HowTo/Configure/Consensus-Protocols/Raft.md) doesn't require a specific item in the `config` object.
+
+## Genesis block parameters
+
+The purpose of some genesis block parameters varies depending on the consensus protocol ([IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md),
+[QBFT](../HowTo/Configure/Consensus-Protocols/QBFT.md), [Raft](../HowTo/Configure/Consensus-Protocols/Raft.md),or
+[Clique](../HowTo/Configure/Consensus-Protocols/Clique.md)).
+These parameters include:
+
+* `difficulty`.
+* `extraData`.
+* `mixHash`.
+
+The following table describes the genesis block parameters with the same purpose across all
+consensus protocols.
+
+| Item                | Description                                                                                                                             |
+|---------------------|-:---------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Does the `:` has a reason?
```suggestion
|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
```

Reply:
`:` is [used to align text in cells at rendering](https://www.markdownguide.org/extended-syntax/#alignment)